electsim/main.go

```diff
@@ -0,0 +1,75 @@
+package main
```

Comment:
I don't think we want to merge this :)

Reply:
Sorry, was in a rush and pushed code from other computer.